docs/data/data-grid/filtering/CustomFilterPanelContent.js

```diff
@@ -65,9 +65,6 @@ export default function CustomFilterPanelContent() {
                 size: 'small',
                 sx: { mt: 'auto' },
               },
-              valueInputProps: {
```

Comment:
Not easily, because the value input component can be any thing developer wants.

By default, we set `variant="standard"` in GridFilterInputValue
https://github.com/alexfauquette/material-ui-x/blob/refacto-export-button/packages/grid/_modules_/grid/components/panel/filterPanel/GridFilterInputValue.tsx#L128:L128

To customize them, developers can [edit operators](https://mui.com/components/data-grid/filtering/#create-a-custom-operator). To make them outlined thy should add to the default operators `InputComponentProps: {variant: "outlined"}`